src/Sentry.Maui/Internal/MauiEventsBinder.cs

```diff
@@ -132,7 +132,14 @@ internal void OnBreadcrumbCreateCallback(BreadcrumbEvent breadcrumb)
             breadcrumb.Sender,
             breadcrumb.EventName,
             UserType,
-            UserActionCategory
+            UserActionCategory,
+            extra =>
+            {
+                foreach (var (key, value) in breadcrumb.ExtraData)
+                {
+                    extra[key] = value;
```

Comment:
should we keep a map instead then? it's odd we have an array but that'll be converted into a map later

Reply:
Then the most generic thing would be `IEnumerable<KeyValuePair<string, string>>? = null`.

Reply:
Yeah, it's not quite as nice to use/call as with tuples (not sure why `KeyValuePair<TKey, TValue>` doesn't just provide an implicit conversion from `(TKey key, TValue value)`) but probably the best choice for internal storage. 

I'll add an overload that so that calling methods can pass tuples or kvps.